com.varnamproject.Varnam.json

```diff
@@ -0,0 +1,171 @@
+{
+    "app-id": "com.varnamproject.Varnam",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "varnam-editor",
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--filesystem=~/.local/share/varnam"
+    ],
+    "modules": [
+        {
+            "name": "ruby",
+            "config-opts": [
+                "--disable-install-doc"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://cache.ruby-lang.org/pub/ruby/2.6/ruby-2.6.3.tar.gz",
+                    "sha256": "577fd3795f22b8d91c1d4e6733637b0394d4082db659fccf224c774a2b1c82fb"
+                }
+            ]
+        },
+        {
+            "name": "rubygems",
+            "buildsystem": "simple",
+            "build-commands": [
+                "gem install --local ffi-1.11.1.gem --no-document"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://rubygems.org/gems/ffi-1.11.1.gem",
+                    "sha256": "90ae2a7fe7ac4c0af30650fba8b75d93ff3611803c4937248118797ae5edbb1a"
+                }
+            ]
+        },
+        {
+            "name": "varnam",
+            "buildsystem": "cmake-ninja",
+            "builddir": false,
+            "post-install": [
+                "export VARNAM_SUGGESTIONS_DIR=/etc/varnam",
+                "./varnamc --compile schemes/ml",
+                "./varnamc --compile schemes/hi",
+                "./varnamc --compile schemes/ta",
+                "./varnamc --compile schemes/te",
+                "./varnamc --compile schemes/bn",
+                "cmake -P cmake_install.cmake"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/subins2000/libvarnam.git",
```

Comment:
Have your changes been merged upstream by any chance?

Reply:
Sadly no, the maintainer of `libvarnam` has been very busy. The PR is still open, and hopefully it will be merged soon.